File: FromSlides/C/23_omp_sections.c
#include <stdio.h>
#include <omp.h>

int main()
{
  printf("Hello in different sections\n");
#pragma omp parallel
  {
#pragma omp sections
    {
    #pragma omp section
    printf("Hello %i\n", omp_get_thread_num());
    #pragma omp section
    printf("Olá %i\n",omp_get_thread_num());
    #pragma omp section
    printf("Hola %i\n",omp_get_thread_num());
    #pragma omp section
    printf("Heghlu'meH QaQ jajvam %i\n",omp_get_thread_num());
    }
  }
  return 0;
}

File: FromSlides/Fortran/23_omp_sections.f90
PROGRAM hello
  USE OMP_LIB
  IMPLICIT NONE
  PRINT *, "Hello in different sections"
!$OMP PARALLEL
!$OMP SECTIONS
    !$OMP SECTION
    PRINT *, "Hello ", omp_get_thread_num()
    !$OMP SECTION
    PRINT *, "Olá ", omp_get_thread_num()
    !$OMP SECTION
    PRINT *, "Hola", omp_get_thread_num()
    !$OMP SECTION
    PRINT *, "Heghlu'meH QaQ jajvam", omp_get_thread_num()
!$OMP END SECTIONS
!$OMP END PARALLEL
END PROGRAM hello
